src/Framework/ChangeWaves.cs

```diff
@@ -27,7 +27,8 @@ internal class ChangeWaves
         internal static readonly Version Wave16_10 = new Version(16, 10);
         internal static readonly Version Wave17_0 = new Version(17, 0);
         internal static readonly Version Wave17_2 = new Version(17, 2);
```

Comment:
I talked briefly with Marc about this yesterday, and we decided we should keep it. Not having it would add to confusion, including around at which point we should "retire" older change waves, and it provides something else people can choose even if it's equivalent to 17_0.